ios/sdk/module/localstorage/HippyAsyncLocalStorage.m

```diff
@@ -313,6 +307,34 @@ - (NSDictionary *)_writeEntry:(NSArray<NSString *> *)entry changedManifest:(BOOL
     return errorOut;
 }
 
+- (NSString *)_readFileFromPath:(NSString *)filePath key:(NSString *)key error:(NSDictionary **)errorOut {
```

Comment:
[suggestion]
'errorOut' is better to be defined as 'NSError' type.  Change the implementation of 'HippyMakeError' to return 'NSError' instance instead of 'NSDictionary'.